source/whats-new.txt

```diff
@@ -22,6 +23,28 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 {+driver-short+} release include:
+
+- Running ``MongoClient.connect()`` is now optional when connecting to your MongoDB instance
+- Use Zstandard compression when communicating with your MongoDB instance
```

Comment:
```suggestion
- Ability to  compress messages with the ``Zstandard`` compression algorithm
```

Also, link to this page since I made it for this release.

Reply:
Committed suggestion + added link